dep/proto/hwsc_file_svc_pb2_grpc.py

```diff
@@ -1,7 +1,7 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 import grpc
 
-import hwsc_file_svc_pb2 as hwsc__file__svc__pb2
+from . import hwsc_file_svc_pb2 as hwsc__file__svc__pb2
 
```

Comment:
This file should be edited because it is a compiled file base on line 1. We should move all the pb files relative to the [service](https://github.com/faraonc/hwsc-file-svc/tree/master/service)